color.go

```diff
@@ -0,0 +1,54 @@
+package termui
+
+import "github.com/fatih/color"
+
+// These mirror their equivalent functions in ui.go
+// Unfortunately since Print is already variadic,
+// we can't have an elegant "...color.Attribute" argument and have to use a slice
```

Comment:
I thought of that too, but figured it was a little hacky. Maybe ease of use is worth more...

But what if I want to print the value of a  `color.Attribute` :stuck_out_tongue: